store/nodegroup.go

```diff
@@ -146,10 +145,7 @@ func (n *Node) Copy() *Node {
 	copy(platforms, n.Platforms)
 	buildkitdFlags := []string{}
 	copy(buildkitdFlags, n.BuildkitdFlags)
-	driverOpts := map[string]string{}
-	for k, v := range n.DriverOpts {
-		driverOpts[k] = v
-	}
+	driverOpts := maps.Clone(n.DriverOpts)
```

Comment:
This is not 100% the same as it could leave `driverOpts` nil.

Note that `modernize --fix` does these updates automatically.